packages/main/src/plugin/tasks/progress-impl.ts

```diff
@@ -135,6 +135,11 @@ export class ProgressImpl {
       action: this.getTaskAction(options),
     });
 
+    let cancelRequest = false;
+    cancellationToken.onCancellationRequested(() => {
+      cancelRequest = true;
```

Comment:
cancellationToken has a property `isCancellationRequested` instead of declaring a variable `cancelRequest`

Reply:
updated